manim/mobject/geometry.py

```diff
@@ -1390,3 +1391,43 @@ def __init__(self, **kwargs):
 class ArrowSquareFilledTip(ArrowFilledTip, ArrowSquareTip):
     r"""Square arrow tip with filled tip."""
     pass
+
+
+class Cutout(VMobject):
+    """A shape with smaller cutouts.
+
+    Parameters
```

Comment:
It would be good to explicitly mention the case where `mobjects` are not fully included in `main_shape`.

```suggestion
    .. warning::

        Technically, this class behaves similar to a symmetric difference: if
        parts of the ``mobjects`` are not located within the ``main_shape``,
        these parts will be added to the resulting :class:`~.VMobject`.

    Parameters
```